client/src/main/java/io/pravega/client/tables/KeyValueTableConfiguration.java

```diff
@@ -73,7 +81,8 @@ public KeyValueTableConfiguration build() {
             Preconditions.checkArgument(this.partitionCount > 0, "partitionCount must be a positive integer. Given %s.", this.partitionCount);
             Preconditions.checkArgument(this.primaryKeyLength > 0, "primaryKeyLength must be a positive integer. Given %s.", this.primaryKeyLength);
             Preconditions.checkArgument(this.secondaryKeyLength >= 0, "secondaryKeyLength must be a non-negative integer. Given %s.", this.secondaryKeyLength);
-            return new KeyValueTableConfiguration(this.partitionCount, this.primaryKeyLength, this.secondaryKeyLength);
+            Preconditions.checkArgument(this.rolloverSizeBytes >= 0, "rollover size must be non-negative. Given %s.", this.rolloverSizeBytes);
```

Comment:
For consistency, let's use the exact same error message as in StreamConfiguration.